markdown/usage/tutorials/step_by_step_institutional_profile.md

```diff
@@ -0,0 +1,617 @@
+---
+title: Step-by-step guide to writing an institutional profile
+subtitle: Walkthrough on what you need to set up an nf-core institutional profile
+---
+
+## Introduction
+
+nf-core offers a centralised place to store Nextflow configuration profiles that work at an _institutional_ level.
+
+What this means is that you can specify common Nextflow pipeline configurations and options that can be shared across all users of your particular institutional cluster.
+
+nf-core offers two level of profile sharing: institutional and pipeline-specific profiles via [nf-core/configs](https://github.com/nf-core/configs)
+
+- Institutional configuration represent configuration options that apply to users of _all_ nf-core pipelines. These typically define settings regarding the cluster itself, such as the type of scheduler being used, maximum resource limits and so on.
+- Pipeline-specific profiles that represent configuration options that apply to users of a _specific_ nf-core pipelines. These typically define common parameters all users of the pipeline would use, or customise resource requirements for particular processes of that specific pipeline
+
+This walkthrough will guide you through making an _institutional_-level profile. It will:
+
+- Describe commonly useful information that is worth gathering _before_ writing such a profile
+- Go through each component of the profile to describe how this should be written
+- Show how to test such a profile before submitting it to nf-core
+
+## Preparation
+
+We will first go through a list of questions to ask and what information to gather prior writing the profile. Not all questions will be relevant to your cluster, however it should cover a range of common set of cluster specifications.
+
+## Does an institutional profile already exist
+
+First you should check nf-core/configs an institutional config exists! If you see any issues with it, or it's not working for you - contact the person who originally made it.
+
+If no config already exists, continue with this tutorial!
+
+### At what level should the institutional profile be designed
+
+Various institutions have different structures in the way clusters are accessed by different groups.
+
+This is an important consideration to make before starting writing a profile because you need to select a useful and descriptive name.
+
+Sometimes one 'institution' will have multiple 'clusters' with the different names, which may share a common storage space. In this case it might make sense to have one nf-core configuration file for the whole institution, but with multiple _internal_ profiles representing each cluster.
+
+Alternatively, within one 'institution', you may have groups or departments with entirely different and separated clusters with dedicated IT teams. In this case it may make sense to have separate configuration files without a common institutional 'umbrella' name.
```

Comment:
```suggestion
Sometimes one institution will have multiple clusters with the different names, which may share a common storage space. In this case it might make sense to have one nf-core configuration file for the whole institution, but with multiple _internal_ profiles representing each cluster.

Alternatively, within one institution, you may have groups or departments with entirely different and separated clusters with dedicated IT teams. In this case it may make sense to have separate configuration files without a common institutional 'umbrella' name.
```